index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>AI for good</title>
+  <link rel="stylesheet" href="style.css"/>
+</head>
+
+<body>
+  <div class="home-container">
+    <header class="main-header">
+      <nav>
+        <div class="nav-bar">
+          <div class="top-nav-bar">
+            <a href="#"><img class="top-nav-item" src="assets/facebook.png" alt="facebook"></a>
+            <a href="#"><img class="top-nav-item" src="assets/twitter .png" alt="twitter"></a>
+            <a href="#" class="top-nav-item">English</a>
+            <a href="#" class="top-nav-item">My Page</a>
+            <a href="#" class="top-nav-item">Logout</a>
+          </div>
+          <div class="bottom-nav-bar">
+            <a href="index.html"><img class="nav-bar-logo" src="assets/ailogo.png" alt="AI-logo"></a>
+            <div class="nav-bar-links">
+              <a class="bottom-nav-link" href="about.html">About</a>
+              <a class="bottom-nav-link" href="#partner">Partner</a>
+              <a class="bottom-nav-link" href="#">Join</a>
+              <a class="bottom-nav-link" href="#">Sponsor</a>
+              <a class="bottom-nav-link" id="ai" href="#">BUY TICKETS</a>
+            </div>
+          </div>
+          <!--  mobile-menu-->
+          <header class="mobile-head">
+            <nav class="mobile-nav">
+              <div class="mob-menu-links">
+                <img class="logo" src="assets/ailogo.png" alt="AI-logo">
+                <p class="mob-menu-para">Join <strong>India's </strong> vision to support <strong>United Nations</strong> in the field of artificial intelligence</p>
+                <a href="#" class="close-link">x</a>
+                <a class="bottom-nav-link" href="about.html"><strong>About</strong></a>
+                <a class="bottom-nav-link" href="#main"><strong>Program</strong></a>
+                <a class="bottom-nav-link" href="#speakers"><strong>Sponsor</strong></a>
+                <a class="bottom-nav-link enroll" href="#"><strong>Buy Ticket</strong></a>
```

Comment:
The `Buy Tickets` link is not properly aligned to the center to match the design, I suggest you kindly fix that as we are required to stick to the design even as we personalize the project👍.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="350" alt="image" src="https://github.com/the-faizmohammad/conferencePage/assets/64643995/51c858ee-9070-44f9-a4b9-f841f86297c6">|<img width="350" alt="image" src="https://github.com/the-faizmohammad/conferencePage/assets/64643995/adfdc93a-dd18-407d-9443-bd2c6e185e5f">|